s3/pkg/service/multipart.go

```diff
@@ -415,6 +415,7 @@ func (s *s3Service) CompleteMultipartUpload(ctx context.Context, in *pb.Complete
 		Type:             ObjectTypeNormal,
 		Tier:             utils.Tier1,
 		Size:             totalSize,
+		Acl:              &pb.Acl{CannedAcl: "private"},
```

Comment:
Hi @sunfch Do we support other ACL settings when CompleteMltipartUpload?

Reply:
@sfzeng  we still dont support to set acl with http header when to put object with multipart now, so      new object uploaded with multipart only has private acl.  thanks